action.yml

```diff
@@ -54,7 +54,7 @@ inputs:
   github_api_url:
     description: 'URL to the API of your Github Server, only necessary for Github Enterprise customers'
     required: false
-    default: 'https://api.github.com'
+    default: '${{ github.api_url }}'
```

Comment:
Verified this in the documentation here: https://docs.github.com/en/actions/learn-github-actions/contexts#github-context